lib/locale/__init__.py

```diff
@@ -0,0 +1,2 @@
+# This directory has an __init__.py so it can be passed to `extractPackages` in
+# android/app/build.gradle.
```

Comment:
Oh i see. `locale.__file__` is defined now.

Reply:
Why is this needed? Is there no other way to do this on Android? Just curious.. thanks.

Reply:
It was the simplest way to get the translation files extracted from the APK so they could be loaded by `gettext`.

Since lots of Chaquopy users have encountered this kind of problem, I'm planning on changing Chaquopy so that all non-Python files will be extracted automatically the first time their top-level package is imported. Then this workaround will be unnecessary.